format/spec.md

```diff
@@ -927,20 +927,21 @@ These rows must be sorted (in ascending manner with NULL FIRST) by `partition` f
 
 The schema of the partition statistics file is as follows:
 
-| v1 | v2 | Field id, name | Type | Description |
-|----|----|----------------|------|-------------|
-| _required_ | _required_ | **`1 partition`** | `struct<..>` | Partition data tuple, schema based on the unified partition type considering all specs in a table |
-| _required_ | _required_ | **`2 spec_id`** | `int` | Partition spec id |
-| _required_ | _required_ | **`3 data_record_count`** | `long` | Count of records in data files |
-| _required_ | _required_ | **`4 data_file_count`** | `int` | Count of data files |
-| _required_ | _required_ | **`5 total_data_file_size_in_bytes`** | `long` | Total size of data files in bytes |
-| _optional_ | _optional_ | **`6 position_delete_record_count`** | `long` | Count of records in position delete files |
-| _optional_ | _optional_ | **`7 position_delete_file_count`** | `int` | Count of position delete files |
-| _optional_ | _optional_ | **`8 equality_delete_record_count`** | `long` | Count of records in equality delete files |
-| _optional_ | _optional_ | **`9 equality_delete_file_count`** | `int` | Count of equality delete files |
-| _optional_ | _optional_ | **`10 total_record_count`** | `long` | Accurate count of records in a partition after applying the delete files if any |
-| _optional_ | _optional_ | **`11 last_updated_at`** | `long` | Timestamp in milliseconds from the unix epoch when the partition was last updated |
-| _optional_ | _optional_ | **`12 last_updated_snapshot_id`** | `long` | ID of snapshot that last updated this partition |
+| v1 | v2 | v3 | Field id, name | Type | Description |
+|----|----|----|----------------|------|-------------|
+| _required_ | _required_ | _required_ | **`1 partition`** | `struct<..>` | Partition data tuple, schema based on the unified partition type considering all specs in a table |
+| _required_ | _required_ | _required_ | **`2 spec_id`** | `int` | Partition spec id |
+| _required_ | _required_ | _required_ | **`3 data_record_count`** | `long` | Count of records in data files |
+| _required_ | _required_ | _required_ | **`4 data_file_count`** | `int` | Count of data files |
+| _required_ | _required_ | _required_ | **`5 total_data_file_size_in_bytes`** | `long` | Total size of data files in bytes |
+| _optional_ | _optional_ | _required_ | **`6 position_delete_record_count`** | `long` | Count of position deletes across position delete files and deletion vectors |
+| _optional_ | _optional_ | _required_ | **`7 position_delete_file_count`** | `int` | Count of position delete files ignoring deletion vectors |
+|            |            | _required_ | **`13 dv_count`** | `int` | Count of deletion vectors |
```

Comment:
Using `dv` in the key to match what we did for snapshot summaries.

Reply:
Yeah, keeping in the end maybe useful for choosing next schema ID easily. Else user might have to go through each field to see whether next schema is reused.

Reply:
@advancedxy @ajantha-bhat Agreed, I think the original intention was to group it with similar fields but I feel like it's better to optimize this table for future evolution

Reply:
I OK either way. Adding it to the end won't reduce the amount of changes, though. We still need to add the V3 column.

Reply:
Actually, we do group by logic in other places. For instance, `nan_value_counts` that was added much later was placed next to other counts. I feel like it makes sense to group by logic as adding to the end won't reduce the number of changes. We need to add the `v3` column.

Reply:
That's a fair point, I'm good either way

Reply:
@aokolnychyi do we need a new field for `dv_record_count`?

Reply:
@stevenzwu, I don't think so but can be convinced otherwise. From the spec perspective, the content of DVs is position deletes. That's what we store in manifests. Also, we combine the number of position deletes across the old-style position delete files and DVs in snapshot summaries. Therefore, the decision to reuse the existing field for the position delete record count is consistent.

Are there use cases that would benefit if we had `dv_record_count` as an independent field?

Reply:
I am thinking about future deprecation. When position delete file is deprecated in the future, `position_delete_record_count` and `position_delete_file_count` fields can be deprecated and removed.

Reply:
My original thinking was to keep `position_delete_record_count` beyond V3 similar to snapshot summary stats, primarily because I consider DV content as position deletes. In V3, this field will contain a sum of record counts across old-style position delete files and DVs. In V4, we will only contain DVs.

I also didn't plan to deprecate  `position_delete_file_count` as marking it as `0` is easy and indicates that there are no old-style position delete files.

How do you see this, @stevenzwu?

Reply:
> because I consider DV content as position deletes

I can understand this reasoning. 

Is there any value to show separate `position_delete_record_count` and `dv_record_count` to help users understand the state of the table in row-level deletes. Maybe the information can help users to make a decision if it makes sense to convert position delete files to delete vectors?

It also feels more symmetric to have `position_delete_record_count` and `position_delete_file_count` (vs) `dv_record_count` and `dv_count`. 

if we do decide to deprecate `position_delete_file_count` in the future, we would have left with `position_delete_record_count` and `dv_count`, which seems a bit inconsistent in naming.

Reply:
I spent a bit more time thinking about this and would still opt in for not introducing a new field to be consistent with how we handle these counts elsewhere (e.g. `SnapshotSummary`). DVs store position deletes so reusing the existing count is appropriate. My gut tells me it is unlikely the record count in DVs vs position delete files would matter based on our discussions around table maintenance. 

@stevenzwu, what do you think?

Reply:
I see. I agree it is not worthwhile if it would introduce pretty significant disruptions to many parts.

what about renaming `dv_count` to `position_delete_vector_count`?